code/modules/tgui/tgui_window.dm

```diff
@@ -370,20 +370,6 @@
 			client << link(href_list["url"])
 		if("cacheReloaded")
 			reinitialize()
-		if("byondui_update")
```

Comment:
This key gets sent from within TGUI as well, you'll need to remove the sendMessage